package-testing/sdk-test-runner/src/app.ts

```diff
@@ -45,31 +99,38 @@ export default class App {
       0,
     );
 
-    const passes = numTestCases - numFailures;
+    const numSkipped = testSuiteResults.reduce(
+      (prev, cur) => cur.testCases.filter((testCase) => testCase.skipped).length + prev,
+      0,
+    );
+
+    const passes = numTestCases - numFailures - numSkipped;
+
+    log(`*** Test Results for ${this.config.sdkName} *** `);
+    const skippedText = numSkipped > 0 ? ` (${numSkipped} skipped)` : '';
+    log(green(`${passes}/${numTestCases} passed${skippedText}`));
 
-    log('*** Test Results *** ');
-    log(green(`${passes}/${numTestCases} passed`));
     const failureFunc = numFailures > 0 ? red : green;
     log(failureFunc(`${numFailures} failures`));
 
     // Junit support.
-    if (this.junitFile) {
-      this.writeJUnitReport(testSuiteResults, 'A report name', this.junitFile);
+    if (this.config.junitFilePath) {
+      this.writeJUnitReport(testSuiteResults, 'A report name', this.config.junitFilePath);
     }
 
     // Exit 1 if there were test failures
     if (numFailures > 0) {
-      return 1;
+      process.exit(1);
     } else {
-      return 0;
+      process.exit(0);
```

Comment:
other options:
```ts
process.exit(numFailures > 0 ? 1 : 0);
```
```ts
process.exit(Math.max(numFailures, 1));
```
Or may favorite:
```ts
const exitCode = numFailures > 0 ? 1 : 0;
process.exit(exitCode);
```
But what you have is totally fine, just trying to show more ternary love.

Reply:
did some simplifying and ternaryizing